slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -764,6 +778,21 @@ void ParseAndValidateArguments(const vector<string> &args,
         BOOST_LOG_TRIVIAL(info) << "No camera given -> running in offline mode";
         slamService.offlineFlag = true;
     }
+
+    auto delete_processed_data = ArgParser(args, "-delete_processed_data=");
+    slamService.delete_processed_data = !(delete_processed_data == "false");
+
+    if (slamService.offlineFlag) {
```

Comment:
Same here, I think you can error if delete_processed_data is true and offlineFlag is false, since rdk should not pass this combination.